### PR TITLE
Added myself to the web team page

### PR DESCRIPTION
**Pull request by mcp31**

### CODE REVIEW

File: about/our-web-team.php [hunk 1]
```diff
@@ -120,18 +161,53 @@
...
+                                    </li>
+                                </ul>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <!-- Mary Plana -->
+                <div class="cell-lg-12 bio-area">
+                    <div class="range range-sm-middle">
+                        <div class="cell-md-3"><img src="/images/bio-pics/mary-plana.jpg" alt="" width="360" height="404"
```
Comment (Peyton232) [inline about/our-web-team.php:193]: should store image in web-team-pics folder

**Review by Peyton232 — CHANGES_REQUESTED** [review]

**Review by sebastian-king — CHANGES_REQUESTED** [review]
Do you have some kind of editor config set in your IDE which is wrapping the lines after a certain number of characters? If you look at the diff for your PR, there is a significant amount of whitespace changes where new line breaks are introduced.

While it's not strictly a major issue to have whitespace changes, even though it does make it very hard to see any actual changes in a PR, this file is also being worked on by other team members and this would cause a major merge conflict with the refactoring that's going on in PR #101. Are you able to update the PR to only include your changes, without the new lines/whitespace changes across the rest of the file?

Great job on the file compression!!

Also -- you have a couple of spots in your section where it still says "Your-Name" which you need to update :)

File: about/our-web-team.php (hunk 1 above)
Comment (sebastian-king) [reply about/our-web-team.php:193]: He means `/images/web-team-pics/mary-plana.jpg`

**Review by Peyton232 — COMMENTED** [review]
she used the beutify command and it auto indented the whole file

**Comment by sebastian-king** [comment]
> she used the beutify command and it auto indented the whole file

Ah, oof, is there an uglify command to undo it? :)
I'm not really sure of a good way to undo that, you might just have to reset the file and the paste just your section into the file after it's been reset.

**Comment by mcp31** [comment]
Lol yea, I do wish there was an uglify button. I'll reset the file and paste my section

**Review by sebastian-king — CHANGES_REQUESTED** [review]
Awesome!
One more tiny change: can you update the IDs from `Mary-Plana` to `mary-plana`? You can 100% blame Peyton for that one for capitalising `Your-Name` in the template because he's a total numpty ;). @Peyton232 can you fix the template in a separate PR? Or, Mary, you're welcome to quickly fix the template in this PR too.

**Review by sebastian-king — APPROVED** [review]

**Review by Peyton232 — APPROVED** [review]
lgtm

**Pull request merged**